viewer/viewer.js

```diff
@@ -3580,6 +3585,204 @@ app.post('/updateUser/:userId', checkToken, function(req, res) {
   });
 });
 
+app.post('/uploadFragment', checkWebEnabled, fragmentUpload, function(req, res) {
+  Db.getUser(req.user.userId, function(err, user) {
```

Comment:
yes any method that has error(), either needs the function error defined or do the res.send yourself